### DISCUSSION TITLE
Sistema de feedback para palestras

### DISCUSSION THREAD

**Discussion by jordaomoraes** [diego3g/rsxp-2023#31] [Ideas]
Salve pessoal, 
Estive pensando em implementarmos uma feature no app onde será possível dar um feedback sobre sobre o conteúdo apresentado por cada speaker.
Podemos discutir se o feedback pode ser visível para toda comunidade ou apenas para a equipe da Rocket.

Faz sentido, o que acham?

**Comment by CassianoJunior** [comment]
Fala @jordaomoraes, acredito que essa feature seja implementada só depois, o intuito dessa primeira versão é mais voltada pra o que acontece antes do evento.
Mas isso com certeza é uma boa ideia pra futuras versões.

**Reply by diego3g** [reply]
Isso!

**Comment by andreghisleni** [comment]
Adorei a idéia e acho que seria intereçante, mas acho que poderiamos deixar a disposição inicialmente dos ouvintes (no caso os participantes) se querem que todos vejam o feedback ou só a equipe da rocket e o speaker
